Dockerfile

```diff
@@ -1,16 +1,23 @@
-FROM alpine
+FROM golang:alpine3.13 AS builder
```

Comment:
What is the reason to pin alpine releases?

Reply:
Reproducible builds and best practice. But here also: Not really needed, so can be reverted if preferred in the original way.